arbor/morph/embed_pwlin.cpp

```diff
@@ -53,6 +53,32 @@ double integrate(const branch_pw_ratpoly<p, q>& f, unsigned bid, const pw_consta
     return accum;
 }
 
+template <unsigned p, unsigned q>
+double integrate(const branch_pw_ratpoly<p, q>& f, mcable c, const pw_constant_fn& g) {
+    msize_t bid = c.branch;
+    double accum = 0;
+
```

Comment:
Yep, good point. I was following a keep it simple for now strategy, but we can certainly revisit it. Would you like a comment so it's marked for later?

Reply:
Agreed, let's not over complicate things. I think a comment would be good, so that one of us is spelunking through there looking for the cause of a performance regression we can save some time.